scripts/initialize.sh

```diff
@@ -21,6 +21,10 @@ else
   brew install watchman
 fi
 
+echo "installing correct pnpm version globally (needed for git hooks)"
+nvm i v18.16.0
+npm install -g pnpm@8.14.3
```

Comment:
I'm creating a v18 instance inside `nvm` here and then installing `pnpm` globally for our target version so that it can be mounted and accessed by the githooks.